.github/issue-branch.yml

```diff
@@ -0,0 +1,43 @@
+branchName: 'issue-${issue.number%4}'
+mode: auto
+gitReplaceChars: '/<>[]'
+gitSafeReplacementChar: '-'
+autoLinkIssue: true
+autoCloseIssue: true
+openDraftPR: true
+prSkipCI: true
+copyIssueDescriptionToPR: true
+copyIssueLabelsToPR: true
+copyIssueAssigneeToPR: true
+copyIssueProjectsToPR: true
+copyIssueMilestoneToPR: true
+conventionalPrTitles: true
+conventionalStyle: gitmoji
+commentMessage: 'Branch ${branchName} created for issue: ${issue.title}'
+conventionalLabels:
+  fix:
+    bug: '🐛'
+    dependencies: '⬆️'
+    security: '🔒'
+  feat:
+    enhancement: '✨'
+    new-stuff: '🚀'
+  build:
+    build: '🔧'
+  chore:
+    chore: '♻️'
+  ci:
+    ci: '👷'
+  docs:
+    documentation: '📝'
+  style:
+    style: '💎'
+  refactor:
+    refactor: '♻️'
+  perf:
+    performance: '⚡️'
+  test:
+    test: '✅'
+  breaking:
+    breaking-change: '💥'
+    breaking change: '💥'
```

Comment:
Guessing we don't need two of these?

Reply:
its a safety net for the case where one does not have a `-` its their recommended setup